### DISCUSSION TITLE
When serverpod cloud is launching?

### DISCUSSION THREAD

**Discussion by ps6067966** [serverpod/serverpod#2444] [General]
So excited for serverpod cloud.

**Comment by SandPod** [comment]
We are aiming for our first beta tester this autumn. We will soon post instructions on how to enroll to as a tester.

**Comment by iampopal** [comment]
Any Updates

**Reply by SandPod** [reply]
We are working hard to get the service up and running.
Right now you can sign up to participate in the private beta here https://t.co/VgZ9HNRoRt.

**Reply by iampopal** [reply]
I have signed up. Can't wait to check it out.
